Code/src/main/java/nl/utwente/viskell/ghcj/GhciSession.java

```diff
@@ -37,7 +37,7 @@
     private Evaluator ghci;
 
     /** Gets filled with a HaskellCatalog instance when ghci is ready. */
-    private HaskellCatalog catalog;
+    private static HaskellCatalog catalog;
```

Comment:
Are you sure this is right? I think that there could potentially be multiple GhciSessions with different catalogs.